editoast/src/generated_data/operational_point.rs

```diff
@@ -70,3 +72,25 @@ impl GeneratedData for OperationalPointLayer {
         Ok(())
     }
 }
+
+#[derive(QueryableByName)]
+struct OperationalPoint {
+    #[diesel(sql_type = Jsonb)]
+    geo: diesel_json::Json<GeoJsonPoint>,
+}
+
+impl OperationalPointLayer {
+    pub async fn get(conn: &mut DbConnection, ids: &[&str]) -> Result<Vec<GeoJsonPoint>> {
+        Ok(sql_query(
+            "SELECT ST_AsGeoJSON(ST_Transform(geographic, 4326))::jsonb AS geo
+                FROM infra_layer_operational_point
+                WHERE obj_id = ANY($1)",
```

Comment:
Hm, I suppose this might now be in-order? Maybe we better return a `HashMap` from this function.

Reply:
All fixed!